_data/menus.yml

```diff
@@ -0,0 +1,11 @@
+---
+main:
```

Comment:
@peckjon I wanted to get your input on whether this is a reasonable setup for you to use: I've added [jekyll-menus](https://github.com/forestryio/jekyll-menus) to handle menu data.

In this file,`main` lists the root menu items (just the dev center, api docs, and a link to algorithmia.com), and each item that has a submenu gets an identifier. (explanatory comment 1/2)